components/brave_new_tab_ui/containers/newTab/settings/topSites.tsx

```diff
@@ -16,11 +16,18 @@ import { getLocale } from '../../../../common/locale'
 interface Props {
```

Comment:
weird, just a note that this file should be in `/components` and not `/containers` since it's a straightforward React pure component and doesn't call any external API or read directly from the redux store.

Reply:
Outside the scope of this PR: I think all of the settings components would be better if they actually read from state and triggered actions. Right now, all the logic seems to be in `components\brave_new_tab_ui\containers\newTab\index.tsx` and the raw value (true/false) and the handler (toggle value) are passed through settings component to the child components.

Although we could move to `/components`, it seems that relying on this is sloppy. I'd advocate for doing them properly with state/actions when we can and keeping in place 😄 WDYT?

Reply:
Well we would want as many UI items as possible to be "dumb" components that don't rely on state specifics and just deal with displaying data and sending actions. So we could put these in /components and make a new /containers/settings that handled all the data retrieval and actions and passed them on to the settings-specific UI components. That's how I think it would be nicely organized anyway.

Reply:
Leaving as-is for now; but that would be a good follow up